web/styles/popovers.css

```diff
@@ -1098,20 +1126,36 @@ ul {
     }
 }
 
-.personal-menu-actions {
+.navbar-dropdown-menu {
     display: flex;
     flex-flow: column nowrap;
+    margin: 0;
+    padding: 0;
+    border: solid 1px var(--color-border-dropdown-menu);
+    background-color: var(--color-background-dropdown-menu);
+    max-height: 85vh;
+    min-width: 230px;
+    overflow-x: hidden;
+    user-select: none;
+    border-radius: 6px;
+    box-shadow: var(--box-shadow-navbar-dropdown-menu);
 
-    .list {
+    .simplebar-content {
+        min-width: max-content;
+    }
+
+    & ul {
         margin: 0;
         padding: 4px 0;
         list-style: none;
-        border-bottom: solid 1px var(--color-border-popover-menu);
         background: none !important;
+    }
 
-        &:last-child {
-            border-bottom: none;
-        }
+    .navbar-dropdown-divider {
```

Comment:
If we provide inner `<ul>` elements for each cluster of items in the menu, this could be expressed way better as just a border.

Reply:
Done.